.github/workflows/daily_common.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
     name: '${{ matrix.platform.os }}-${{ matrix.cpu }} (Nim ${{ matrix.nim.branch }})'
     runs-on: ${{ matrix.platform.builder }}
-    continue-on-error: ${{ matrix.nim.branch == 'devel' || matrix.nim.branch == 'version-2-0' }}
+    if: success() || failure() # Continue jobs even if previous jobs failed, but not if cancelled.
```

Comment:
I'm not 100% sure this change will fix the problem. I just see that as a way to write `continue-on-error` differently. But I dig into this a bit too much and I think removing this line will solve the issue.
With `fail-fast: false` defined in the strategy up above, the jobs will continue on success and failure, even without this line (or the `continue-on-error`).

Reply:
I'm also not completely sure. But according to the post I mentioned in the description it apparently will do what it suggests. 

I could set the daily jobs to run on PR so we can get the results here, now that I think about it.

Reply:
It seems it works, as the jobs have been marked as failed? I can't say for sure, though.

Reply:
Could you try without this line, just for my own sanity? :smile:

Reply:
Besides that, like you said, it seems it worked. Just remove the `on pull_request` and I approve it.

Reply:
Here the checks show as failed, but if you go check the `Actions` tab you'll see a difference between them. The workflows run with `continue-on-error` show as green tick (even if they failed), while the ones with the `if: success() || failure()` show as failed; which is the intended behaviour.

Reply:
Oh, I see. Doing that :)

Reply:
It seems to work without it.

Reply:
True. Shall we go for this?